generator/templates/src/components/Component/Component.spec.tsx

```diff
@@ -1,40 +1,50 @@
 import * as React from 'react';
+import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
+import { isUniEnzymeTestkitExists } from 'wix-ui-test-utils/enzyme';
+import { isUniTestkitExists } from 'wix-ui-test-utils/vanilla';
+import { mount } from 'enzyme';
+import { TPAComponentsWrapper } from '../../test/utils';
 import { {%componentName%}DriverFactory } from './{%ComponentName%}.driver';
 import { {%ComponentName%} } from './';
-import { createDriverFactory } from 'wix-ui-test-utils/driver-factory';
-import { isEnzymeTestkitExists } from 'wix-ui-test-utils/enzyme';
-import { isTestkitExists } from 'wix-ui-test-utils/vanilla';
-import { mount } from 'enzyme';
 import { {%componentName%}TestkitFactory } from '../../testkit';
 import { {%componentName%}TestkitFactory as enzyme{%ComponentName%}TestkitFactory } from '../../testkit/enzyme';
 
 describe('{%ComponentName%}', () => {
-  const createDriver = createDriverFactory({%componentName%}DriverFactory);
+  const createDriver = createUniDriverFactory({%componentName%}DriverFactory);
+
+  it('should render', async () => {
+    const driver = createDriver(<{%ComponentName%} buttonText="Click Me" />);
+    expect(await driver.exists()).toBe(true);
+  });
 
-  it('should render', () => {
-    const value = 'hello!';
+  it('should use mobile design', async () => {
     const driver = createDriver(
-      <{%ComponentName%} dataHook="some hook" buttonText="Click Me" />,
+      TPAComponentsWrapper({ mobile: true })(<{%ComponentName%} />),
     );
-    //make an assertion
+    expect(await driver.isMobile()).toBe(true);
   });
 
   describe('testkit', () => {
-    it('should exist', () => {
+    it('should exist', async () => {
       expect(
-        isTestkitExists(<{%ComponentName%} />, {%componentName%}TestkitFactory, {
+        await isUniTestkitExists(<{%ComponentName%} />, {%componentName%}TestkitFactory, {
           dataHookPropName: 'data-hook',
         }),
       ).toBe(true);
     });
   });
 
   describe('enzyme testkit', () => {
-    it('should exist', () => {
+    it('should exist', async () => {
       expect(
-        isEnzymeTestkitExists(<{%ComponentName%} />, enzyme{%ComponentName%}TestkitFactory, mount, {
-          dataHookPropName: 'data-hook',
-        }),
+        await isUniEnzymeTestkitExists(
```

Comment:
just fyi, eventually `should exist` tests will also be automated with wuf. so for now it's ok to add them to each component but the goal is to automate it

it is already automated in wsr, need to migrate and generalize the script

so it's just fyi, it'll happen some time :)